test/cadet/accounts/accounts_test.exs

```diff
@@ -134,73 +133,46 @@ defmodule Cadet.AccountsTest do
         user: user
       })
 
-      assert {:ok, user} == Accounts.sign_in("e012345", @token)
+      assert {:ok, user} == Accounts.sign_in("e012345", "TOM", @token)
     end
 
     test "invalid token" do
       use_cassette "accounts/sign_in#2" do
-        assert {:error, :bad_request} == Accounts.sign_in("A0123456", "t0k3n")
+        assert {:error, :bad_request} == Accounts.sign_in("A0123456", "TOM", "t0k3n")
       end
     end
 
     test "invalid nusnet id" do
       use_cassette "accounts/sign_in#3" do
-        assert {:error, :internal_server_error} == Accounts.sign_in("", @token)
+        assert {:error, :internal_server_error} == Accounts.sign_in("", "TOM", @token)
       end
     end
   end
 
   describe "sign in with unregistered user gets the right roles" do
-    test ~s(user has permission "O") do
+    test ~s(user has full access) do
       use_cassette "accounts/sign_in#4", custom: true do
-        assert {:ok, user} = Accounts.sign_in("e012345", @token)
+        assert {:ok, user} = Accounts.sign_in("e012345", "TOM", @token)
         assert %{role: :admin} = user
       end
     end
 
-    test ~s(user has permission "F") do
+    test ~s(user has create access) do
       use_cassette "accounts/sign_in#5", custom: true do
-        assert {:ok, user} = Accounts.sign_in("e012345", @token)
-        assert %{role: :admin} = user
-      end
-    end
-
-    test ~s(user has permission "M") do
-      use_cassette "accounts/sign_in#6", custom: true do
-        assert {:ok, user} = Accounts.sign_in("e012345", @token)
-        assert %{role: :staff} = user
-      end
-    end
-
-    test ~s(user has permission "R") do
-      use_cassette "accounts/sign_in#7", custom: true do
-        assert {:ok, user} = Accounts.sign_in("e012345", @token)
+        assert {:ok, user} = Accounts.sign_in("e012345", "TOM", @token)
         assert %{role: :staff} = user
       end
     end
 
-    test ~s(user has permission "S") do
-      use_cassette "accounts/sign_in#8", custom: true do
-        assert {:ok, user} = Accounts.sign_in("e012345", @token)
-        assert %{role: :student} = user
-      end
-    end
-
-    test ~s(user has unknown permission "A") do
-      use_cassette "accounts/sign_in#9", custom: true do
-        assert {:error, :bad_request} = Accounts.sign_in("e012345", @token)
-      end
-    end
-
     test ~s(user cs1101s module is inactive) do
```

Comment:
U sure that inactive mods has that code? Maybe check with LumiNUS? Or like I said earlier check the module's semester instead